pkg/infrastructure/vsphere/clusterapi/import.go

```diff
@@ -228,9 +226,10 @@ func findAvailableHostSystems(ctx context.Context, clusterHostSystems []*object.
 		// if distributed port group the cast will fail
 		networkFound := isNetworkAvailable(networkObjectRef, hostSystemManagedObject.Network)
 		datastoreFound := isDatastoreAvailable(datastore, hostSystemManagedObject.Datastore)
+		isUsable := hostSystemManagedObject.Runtime.PowerState != types.HostSystemPowerStatePoweredOff && hostSystemManagedObject.Runtime.PowerState != types.HostSystemPowerStateStandBy
```

Comment:
@Nabsku thank you for the PR!

`isUsable` feels a bit vague to me what that boolean is. Can we rename it closer to what it represents? Something related to powerstate?

Reply:
Of course! I’ll do that first thing in the morning.

Reply:
I changed the variable name to closer reflect what it does and pulled the Maintenance Mode check into it as well. Hope thats fine for you!